module/account/features/account-not-authorized.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Account module
```

Comment:
The file should be not-authenticated ;) So does the others with similar tests ofc